servers/physics_server_2d.h

```diff
@@ -828,6 +837,7 @@ VARIANT_ENUM_CAST(PhysicsServer2D::BodyDampMode);
 VARIANT_ENUM_CAST(PhysicsServer2D::BodyState);
 VARIANT_ENUM_CAST(PhysicsServer2D::CCDMode);
 VARIANT_ENUM_CAST(PhysicsServer2D::JointParam);
+VARIANT_ENUM_CAST(PhysicsServer2D::JointState);
```

Comment:
You will need to register the values of of this enum in `_bind_methods` of `PhysicsServer2D`,

Reply:
You need to bind them with `BIND_ENUM_CONSTANT` like you did in the other PR